workspaces/resource-optimization/package.json

```
@@ -20,6 +20,7 @@
     "fix": "backstage-cli repo fix",
     "lint": "backstage-cli repo lint --since origin/main",
     "lint:all": "backstage-cli repo lint",
+    "format": "prettier --write \"{packages,plugins}/**/*.{js,jsx,ts,tsx,css,scss,md,mjs,yaml}\"",
```

Prettier is already configured to treat the files in the packages and plugins folder, including those files with supported file extensions. 
```suggestion
    "format": "prettier --write .",
```